dlt/destinations/qdrant/qdrant_client.py

```diff
@@ -0,0 +1,404 @@
+        """
+
+        # A straight-forward method named get_fastembed_vector_params() exists in the qdrant_client package.
+        # But, it generates a named vector with the model name as the vector name. But, we need an unnamed vector.
+        embeddings_size, distance = self.db_client._get_model_params(
+            model_name=self.db_client.embedding_model_name)
+        vectors_config = models.VectorParams(
+            size=embeddings_size, distance=distance)
+
+        self.db_client.create_collection(
+            collection_name=full_collection_name, vectors_config=vectors_config)
+
+    def _create_point(self, obj: Dict[str, Any], collection_name: str) -> None:
+        """Inserts a point into a Qdrant collection without a vector.
+
+        Args:
+            obj (Dict[str, Any]): The arbitrary data to be inserted as payload.
+            collection_name (str): The name of the collection to insert the point into.
+        """
+        self.db_client.upsert(collection_name, points=[
+            models.PointStruct(
+                id=str(uuid.uuid4()),
+                payload=obj,
+                vector={},
+            )])
+
+    def drop_storage(self) -> None:
+        """Drop the dataset from the Qdrant instance.
+
+        Deletes all collections in the dataset and all data associated.
+        Deletes the sentinel collection.
+
+        If dataset name was not provided, it deletes all the tables in the current schema
+        """
+        collections = self.db_client.get_collections().collections
+        collection_name_list = [collection.name
+                           for collection in collections]
+
+        if self.dataset_name:
+            prefix = f"{self.dataset_name}{self.config.dataset_separator}"
+
+            for collection_name in collection_name_list:
+                if collection_name.startswith(prefix):
+                    self.db_client.delete_collection(collection_name)
+        else:
+            for collection_name in self.schema.tables.keys():
+                if collection_name in collection_name_list:
+                    self.db_client.delete_collection(collection_name)
+
+        self._delete_sentinel_collection()
+
+    def initialize_storage(self, truncate_tables: Iterable[str] = None) -> None:
+        if not self.is_storage_initialized():
+            self._create_sentinel_collection()
+        elif truncate_tables:
+            for table_name in truncate_tables:
+                qualified_table_name = self._make_qualified_collection_name(
+                    table_name=table_name)
+                if self._collection_exists(qualified_table_name):
+                    continue
+
+                self.db_client.delete_collection(qualified_table_name)
+                self._create_collection(full_collection_name=qualified_table_name)
+
+    def is_storage_initialized(self) -> bool:
+        self._collection_exists(self.sentinel_collection)
```

Comment:
```suggestion
        return self._collection_exists(self.sentinel_collection)
```
I think that is the cause of the failed pipeline I mentioned.